gitevanhsu/book-reviews
# Fixbugs

**Description**
1. Fixed when member change rating, the member's review will reflash bug.
2. Fixed member's name is too long, the page will broken bug.
3. The store to bookshelf and switch group、review page's area fixed on page, member can simply use.
4. Combine review's comment and friend request together.
5. Combine search page and books page together.
6. Unify all page's font size and button gap.
7. Move friend list to header.

**Test Plan**
1. Try leave rating and review, edit it rating、title or content make sure other part is fine. also remove is ok.
2. Use other account to leave comment under specific member and sent friend request, It show on header.
3. Switch review page and group page by using button fixed on page.
4. Click one of the friend on friend list to the friend's personal page.
